crates/core/src/kernel/transaction/mod.rs

```diff
@@ -619,14 +619,21 @@ impl<'a> std::future::IntoFuture for PreparedCommit<'a> {
             let commit_or_bytes = this.commit_or_bytes;
 
             if this.table_data.is_none() {
+                tracing::debug!("committing initial table version 0");
```

Comment:
:wave: in a lot of places we actually just import `use tracing::*;` which means this module prefix isn't necessary.

IMHO it's reasonable for us to pull `tracing::*` into most modules :smile:

Reply:
Just imported with `use tracing::*;`, if you found it's ok for readability.
I didn't want to mix `log` and `tracing` crates, but it seems reasonable for now! We can tighten the imports or add explicit bridging later if the project grows.